modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/odata/entity/v1_0/SearchResultEntityModel.java

```diff
@@ -45,12 +45,14 @@ public SearchResultEntityModel() {
 				"description",
 				locale -> Field.getSortableFieldName(
 					LocalizationUtil.getLocalizedName(
-						Field.DESCRIPTION, LocaleUtil.toLanguageId(locale)))),
+						Field.DESCRIPTION, LocaleUtil.toLanguageId(locale))),
```

Comment:
Having "description" at all might actually be a mistake. I think it should be removed because I don't see it being used as a filter in headless-delivery-impl entities and not sure if it makes much sense.

if @lipusz agrees, let's remove it and update docs?

Reply:
You mean not to return `description` field at all in the response?

Reply:
No no, this is about the fields available for "filter". 
Taking a look at headless-delivery-impl, I don't see this field available in other entities . 
Might be in some places, though, but I'm not sure how useful this is. And we can always add it later, if requested.

Reply:
Since it's an analyzed text field, would it even work as a `filter`?

Reply:
Just asking because all the other filter fields seem to be keyword/numeric or date types.

Reply:
I'm okay to remove the `description` field from the available `filter` fields. :+1: I'll update the confluence article and notify Russ.

Reply:
> Since it's an analyzed text field, would it even work as a filter?
No, but I think lambdas as filters do a DB query also. So, something like contains('description', '...') could still work. Not sure about this though. The bottom line was, that it's not used in the headless-delivery-impl reference module.